Home_work/Знакомство с языком Python/Семинар_7/work_with_file.py

```diff
@@ -0,0 +1,145 @@
+import glob
+import random
+import csv
+import view as v
+import work_with_strings as wws
+
+
+def initializing_variable():
+    global data_file
+    data_file = ''
+    global name_download_file
+    name_download_file = ''
+    global max_id
+    max_id = ''
+    return data_file
+
+
+def search_file_and_download(name_file):
+    if glob.glob(f'{name_file}.csv'):
+        contain_data_file = read_file(name_file)
+        v.contain_data_file_true()
+        global name_download_file
+        name_download_file = name_file
+        return contain_data_file
+    else:
+        v.error_null_file_name()
+        return search_file_and_download(v.print_enter())
+
+
+def read_file(name_file):
+    with open(f'{name_file}.csv', 'r') as data:
+        temp = data.read().split()
```

Comment:
При добавлении данных здесь возникает ошибка.
![image](https://user-images.githubusercontent.com/49741194/204570722-0c2fa451-299f-4009-84c5-71be8aa44c43.png)
Это все из за того, что вы не указали кодировку во время чтения, но при записи она есть.
```suggestion
    with open(f'{name_file}.csv', 'r', encoding='utf-8') as data:
        temp = data.read().split()
```